admin/build_targets/ubuntu-16.04/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright ClusterHQ Inc. See LICENSE file for details.
+#
+# A Docker image for building packages in a clean Ubuntu 15.10 build
```

Comment:
Can you please update the comment from ``Ubuntu 15.10`` to ``Ubuntu 16.04``? Thanks!